frontend/assets/stylesheets/base/_helpers.scss

```diff
@@ -7,6 +7,7 @@
     display: none !important;
 }
 
+.is-visually-hidden,
```

Comment:
This isn't any better than using `u-h` see: https://github.com/guardian/membership-frontend/blob/master/docs/FRONTEND.md#state-classes